system/repositories.rs

```diff
@@ -74,21 +74,23 @@ pub mod user_repository {
         function_manager: &FunctionManager,
         query_manager: &QueryManager,
         user: &User,
-        credential: &Credential,
+        credentials: &Credential,
```

Comment:
Should not the struct be `Credentials` instead of `Credential` as well?

Reply:
Yes, I'm planning to do it in a follow up PR after we've finished with the release since there are many places where the rename should take place.

As you've seen from this PR and also the PRs in driver, I've only introduced the new name `Credentials` on places that are public facing and should not change after release, namely, the TypeDB Driver API and this database schema.